src/main/kotlin/com/dansplugins/factionsystem/claim/MfClaimService.kt

```diff
@@ -60,6 +60,24 @@ class MfClaimService(private val plugin: MedievalFactions, private val repositor
         return false
     }
 
+    // Checks whether a set of chunks has at least one chunk that is adjacent to an existing claim. Works across multiple worlds.
+    fun claimIsAdjacent(chunks: Set<Chunk>, id: MfFactionId): Boolean
+    {
+        for (chunk in chunks)
```

Comment:
You can achieve a bit more brevity with something like chunks.all, which checks a predicate against all members of a collection